backend/test/Helpers/ServiceHelper.cs

```diff
@@ -11,21 +12,101 @@ public static class ServiceHelper
         /// </summary>
         /// <param name="helper"></param>
         /// <returns></returns>
-        public static IPimsService CreatePimsService (this TestHelper helper)
+        public static IPimsService CreatePimsService(this TestHelper helper)
         {
-            var service = new Mock<IPimsService> ();
-            var parcelService = new Mock<IParcelService> ();
-            service.Setup (m => m.Parcel).Returns (parcelService.Object);
-
-            var lookupService = new Mock<ILookupService> ();
-            service.Setup (m => m.Lookup).Returns (lookupService.Object);
-
-            helper.AddSingleton (service);
-            helper.AddSingleton (service.Object);
-            helper.AddSingleton (parcelService);
-            helper.AddSingleton (parcelService.Object);
-            helper.AddSingleton (lookupService);
-            helper.AddSingleton (lookupService.Object);
+            var service = new Mock<IPimsService>();
```

Comment:
Looks like something happened with the whitespace here.

Reply:
I'm blind, because I'm not seeing it...

Reply:
It's all the spaces between method names and the parentheses - 
helper.AddSingleton (lookupService) -> helper.AddSingleton(lookupService)